src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/utils.ts

```diff
@@ -15,3 +15,14 @@ export const tomUtgiftRad = (): Utgift => ({
     fom: '',
     tom: '',
 });
+
+export const lagVedtakRequest = (
+    form: FormState<InnvilgeVedtakForm>
+): InnvilgeVedtakForBarnetilsyn => {
+    return {
+        stønadsperioder: form.stønadsperioder,
+        utgifter: form.utgifter,
+        _type: VedtakType.InnvilgelseBarnetilsyn,
+        resultatType: BehandlingResultat.INNVILGET,
```

Comment:
Disse to er vel ikke med i dto?

Reply:
Det er samme type på både beregning og lagre vedtak. 

I EF så er disse feltene med, men ikke hos oss nå. Ble nevnt i går at vi kanskje må sette de, men vet ikke om de skal det eller ikke. Så foreløpig er de beholdt, men kan slette de fra typen

Reply:
De er i hvert fall ikke med i det beregningsendepunktet som er i main nå, men det gjør ingenting hvis det skal bli sånn